src/sockperf.cpp

```diff
@@ -2307,6 +2310,29 @@ static int parse_client_opt(const AOPT_OBJECT *client_obj) {
                 }
             }
         }
+        if (!rc && aopt_check(client_obj, OPT_TCP_NB_CONN_TIMEOUT_MS)) {
+             if (s_user_params.feedfile_name[0] != '\0' || s_user_params.sock_type == SOCK_STREAM) {
+                const char *optarg = aopt_value(client_obj, OPT_TCP_NB_CONN_TIMEOUT_MS);
+                if (optarg) {
+                    errno = 0;
+                    int value = strtol(optarg, NULL, 0);
+                    if (errno != 0 || value < 0) {
+                        log_msg("'-%d' Invalid TCP connect timeout value: %s",
```

Comment:
`-%d` looks weird here. I suggest to replace it with `--tcp_nb_connect_timeout_ms` since this option does not have a short form

Reply:
Done